simulation_parameters/Constants.cs

```diff
@@ -603,6 +603,7 @@ public static class Constants
     /// </summary>
     public const int MAX_JSON_ERROR_LENGTH_FOR_CONSOLE = 20000;
 
+    public const string SAVE_NAME_BLACKLIST = "<>:\"/\\|?*\0";
```

Comment:
use `System.IO.Path.GetInvalidFileNameChars()`

Reply:
Can't do that https://github.com/Revolutionary-Games/Thrive/issues/2862#issuecomment-980558005